WordPress/src/main/java/org/wordpress/android/models/ReaderTag.java

```diff
@@ -29,49 +29,54 @@ public ReaderTag(String slug,
         // primary key in the tag table)
         if (TextUtils.isEmpty(slug)) {
             if (!TextUtils.isEmpty(title)) {
-                setTagSlug(ReaderUtils.sanitizeWithDashes(title));
+                setmTagSlug(ReaderUtils.sanitizeWithDashes(title));
             } else {
-                setTagSlug(getTagSlugFromEndpoint(endpoint));
+                setmTagSlug(getTagSlugFromEndpoint(endpoint));
             }
         } else {
-            setTagSlug(slug);
+            setmTagSlug(slug);
         }
 
-        setTagDisplayName(displayName);
-        setTagTitle(title);
-        setEndpoint(endpoint);
+        setmTagDisplayName(displayName);
+        setmTagTitle(title);
+        setmEndpoint(endpoint);
         this.tagType = tagType;
     }
 
-    public String getEndpoint() {
-        return StringUtils.notNullStr(endpoint);
+    public String getmEndpoint() {
```

Comment:
this is wrong, right? the getters and setters shouldn't contain the `m`

Reply:
Argh, yes this wrong, well spotted!